src/ogt_vox.h

```diff
@@ -232,6 +232,9 @@
         uint32_t       size_x;        // number of voxels in the local x dimension
         uint32_t       size_y;        // number of voxels in the local y dimension
         uint32_t       size_z;        // number of voxels in the local z dimension
+        float          pivot_x;       // the pivot when applying the ogt_vox_transform data
```

Comment:
can we remove these fields from ogt_vox_model? These are entirely derivable by each client of the library, would like to keep this library as minimal as possible?